src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -486,10 +497,10 @@ public function viewAction(int $orderId, Request $request): Response
         try {
             $this->dispatchHook(
                 'actionGetAdminOrderButtons', [
-                    'controller' => $this,
-                    'id_order' => $orderId,
-                    'actions_bar_buttons_collection' => $backOfficeOrderButtons,
-                ]);
+                'controller' => $this,
+                'id_order' => $orderId,
+                'actions_bar_buttons_collection' => $backOfficeOrderButtons,
+            ]);
```

Comment:
My IDE insisted on refactoring that. Even when I put back the original content it AUTOMATICALLY fixes the indent 😆